### PR TITLE
br: update tikv config for pitr

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/10757
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1581,6 +1581,46 @@ Configuration items related to BR backup.
...
+Configuration items related to log backup.
+
+### `enable` <span class="version-mark">New in v6.2.0</span>
+
++ Determines whether to enable log backup.
++ Default value: `false`
+
+### `file-size-limit` <span class="version-mark">New in v6.2.0</span>
+
++ The size limit on log backup data. Once this limit is reached, data is automatically flushed to external storage.
```
Comment (ran-huang) [inline tikv-configuration-file.md:1595]:
```suggestion
+ The size limit on log backup data. Once this limit is reached, the backup data is automatically flushed to external storage.
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1581,6 +1581,46 @@ Configuration items related to BR backup.
...
++ Default value: 256MB
+
+### `initial-scan-pending-memory-quota` <span class="version-mark">New in v6.2.0</span>
+
++ The quota of cache used for storing incremental scan data.
```
Comment (ran-huang) [inline tikv-configuration-file.md:1600]:
```suggestion
+ The quota of cache used for storing incremental scan data during log backup.
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by joccau — APPROVED** [review]
LGTM

**Pull request merged**